preliz/utils/citations.py

```diff
@@ -0,0 +1,51 @@
+import os
+import re
+
+try:
+    from IPython.display import Markdown, display
+except ImportError:
+    pass
+
+
+def citations(methods=None, show_as="bibtex"):
```

Comment:
This should also work outside a notebook environment

Reply:
In that case, will it return a string of the citations?

Reply:
yes, we should also have an option to write to a file